_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: default
+title: Index management
+parent: Admin UI for index operations in OpenSearch Dashboards
```

Comment:
@ariamarble After discussing with @dagneyb, we agreed that this should be called "index management UI enhancements" and not "Admin UI". Because the term appears so frequently in this topic, can you take a pass at changing the references before I edit? Let me know if you'd like to discuss. Thanks!